app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<h3>Post a Comment</h3>
+<%= form_for [@comment] do |f| %>
+  <%= f.text_field :content %>
+  <%= hidden_field_tag :job_id, @job.id %>
```

Comment:
we talked about in class how if we nest our routes, we wouldn't need to pass hidden input 👍 let me know if you have questions about this!